base/shell/explorer/explorer.cpp

```diff
@@ -167,6 +169,7 @@ StartWithDesktop(IN HINSTANCE hInstance)
         ProcessStartupItems();
         DoFinishStartupItems();
```

Comment:
DoFinishStartupItems now just calls ReleaseStartupMutex.
It could then be just used instead of calling again ReleaseStartupMutex below.

Reply:
This branch is maybe taken and is Finish for what it Started. The mutex might be initialized higher up and always needs release outside the branch.